src/js/components/RangeInput/__tests__/RangeInput-test.tsx

```diff
@@ -138,4 +140,35 @@ describe('RangeInput', () => {
 
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('onWheel should call onChange by default', () => {
+    const onChange = jest.fn();
+    const { getByDisplayValue } = render(
+      <Grommet>
+        <RangeInput min={0} max={10} step={1} value={5} onChange={onChange} />
+      </Grommet>,
+    );
+
+    fireEvent.wheel(getByDisplayValue('5'), {
+      target: {
+        value: '10',
+      },
+    });
+    expect(onChange).toBeCalledTimes(1);
+  });
+
+  test('onWheel overrides should not call onChange', () => {
+    const theme: ThemeType = { rangeInput: { wheel: false } };
+    const onChange = jest.fn();
+    const { getByDisplayValue } = render(
+      <Grommet>
+        <ThemeContext.Extend value={theme}>
+          <RangeInput min={0} max={10} step={1} value={5} onChange={onChange} />
+        </ThemeContext.Extend>
+      </Grommet>,
```

Comment:
```suggestion
    const theme: ThemeType = { rangeInput: { wheel: false } };
    const onChange = jest.fn();
    const { getByDisplayValue } = render(
      <Grommet theme={theme}>
          <RangeInput min={0} max={10} step={1} value={5} onChange={onChange} />
      </Grommet>,
```
Just a minor simplification so we no longer need to import ThemeContext